optimum/exporters/onnx/model_configs.py

```diff
@@ -645,6 +646,102 @@ def outputs(self) -> Mapping[str, Mapping[int, str]]:
         }
 
 
+class CLIPTextOnnxConfig(TextEncoderOnnxConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedConfig.with_args(
+        vocab_size="vocab_size",
+        sequence_length="max_position_embeddings",
+        allow_new=True,
+    )
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return {
+            "input_ids": {0: "batch_size", 1: "sequence_length"},
+        }
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
+        return {
+            "last_hidden_state": {0: "batch_size", 1: "sequence_length", 2: "feature_dim"},
+            "pooler_output": {0: "batch_size", 1: "feature_dim"},
+        }
+
+    def generate_dummy_inputs(self, framework: str = "pt"):
+        dummy_inputs = super().generate_dummy_inputs(framework=framework)
+        if framework == "pt":
+            import torch
+
+            dummy_inputs["input_ids"] = dummy_inputs["input_ids"].to(dtype=torch.int32)
```

Comment:
Why? Dummy inputs are not already in `int32`?

Reply:
`torch.randint` returns a tensor of type `torch.int64` by default